mlflow/tensorflow.py

```diff
@@ -212,10 +218,17 @@ def load_model(model_uri, tf_sess):
                       `Referencing Artifacts <https://www.mlflow.org/docs/latest/tracking.html#
                       artifact-locations>`_.
 
-    :param tf_sess: The TensorFlow session in which to load the model.
-    :return: A TensorFlow signature definition of type:
+
+    :param tf_sess: The TensorFlow session in which to load the model. Since in TensorFlow
```

Comment:
```suggestion
    :param tf_sess: The TensorFlow session in which to load the model. If using TensorFlow version >= 2.0.0, this 
    argument is ignored. If using TensorFlow <2.0.0, if no session is passed to this function, MLflow will attempt to 
    load the model using the default TensorFlow session.  If no default session is available, then the function
    raises an exception.
```

Small wording suggestion (removed the "Since")